### PR TITLE
Serenity-Updates-July-2022

### PR DESCRIPTION
**Pull request by sarahhoffmanakeneo**
Serenity updates and changes to the what's next page in Akeneo help center

### CODE REVIEW

**Review by sarahhoffmanakeneo — COMMENTED** [review]
Please review updates to tailored imports and what's next page

File: content/updates/2022-07/01-TailoredImports.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+pim_announcement_img: ./img/refentities.png
+pim_announcement_alt_img: Reference entities for colors
+pim_announcement_audience:
+- EE
+---
+
+# Tailored Imports
```
Comment (sarahhoffmanakeneo) [inline content/updates/2022-07/01-TailoredImports.md:8]:
```suggestion
# Tailored Imports supports more attribute types
```

File: content/updates/2022-07/01-TailoredImports.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+::: meta-data type="Improvement" features="Productivity" available="July" in="EE" link-to-doc="../articles/tailored-import.html"
+
+Tailored Imports allos you to easily transform, map, and import data files that don't match your Akeneo PIM catalog structure. Now you can map columns in your source data to additional attribute types in your Akeneo PIM. These types include multiple-link Reference Entities and sinlge-link Reference Entities, as well as Asset Collections. This helps by offering smoother mapping and faster importing.
```
Comment (sarahhoffmanakeneo) [inline content/updates/2022-07/01-TailoredImports.md:11]:
```suggestion
Tailored Imports allows you to easily transform, map, and import data files that don't match your Akeneo PIM catalog structure. Now you can map columns in your source data to additional attribute types in your Akeneo PIM. These types include multiple-link Reference Entities and single-link Reference Entities, as well as Asset Collections. This helps by offering smoother mapping and faster importing.
```

File: content/updates/index.json [hunk 3]
```diff
@@ -1,4 +1,9 @@
 {
+  "2022-07": {
+    "title": "July 2022 Serenity updates",
+    "description": "Tailored Imports has increased funcationality by incluidng Reference Entities and Asset Collections, try it out today!",
```
Comment (sarahhoffmanakeneo) [inline content/updates/index.json:4]:
```suggestion
    "description": "Tailored Imports has increased functionality by including Reference Entities and Asset Collections, try it out today!",
```

**Comment by JoshuaAkeneo** [comment]
Per Sarah, this needs to go live. I have verified the information is correct.

**Review by JoshuaAkeneo — APPROVED** [review]

**Review by sarahhoffmanakeneo — COMMENTED** [review]
Typos fixed

**Review by monikaakeneo — APPROVED** [review]

**Pull request merged**